pkgs/applications/audio/vgmstream/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, lib, fetchFromGitHub, cmake, pkg-config
+, mpg123, ffmpeg, libvorbis, libao, jansson
+}:
+stdenv.mkDerivation rec {
+  pname   = "vgmstream";
+  version = "r1050-3738-g7cfa1f29";
```

Comment:
Are the tags continuously updated?

Reply:
That's not a tag, that's a `git describe` on the latest commit at time of submission (also coincidentally the name that's used for tags). Users are expected to use the nightlies from the website.

Reply:
I am not sure how we want to update this automatically. Either we use release tags or unstable-XXXX-XX-XX format.

Reply:
Maybe you want to ask upstream how they think this should be packaged and updated because I have no good idea right now.

Reply:
i've just set it to the latest tag. Releases seemed to be tagged intermittently, so I suggest we just follow those.